test/integration/full/jquery/a11ycheck-object-exclude.js

```diff
@@ -0,0 +1,13 @@
+/*global $ */
+
+describe('jQuery object as a11yCheck context', function () {
+	'use strict';
+	it('should find no violations', function (done) {
+		var target = $('#target')[0];
+		axe.a11yCheck({ exclude: [target] }, {}, function (results) {
+			assert.lengthOf(results.violations, 0, 'violations');
+			assert.lengthOf(results.passes, 11, 'passes');
```

Comment:
@dylanb to confirm this is working as expected, should there be passes for an excluded target that has failures in it (meaning should they run at all)?